inc/aioseop_functions.php

```diff
@@ -1081,3 +1081,80 @@ function aioseop_formatted_date( $date = null, $format = null ) {
 		return apply_filters( 'aioseop_format_date', $formatted_date, $date, $format );
 	}
 }
+
+/**
+ * SVG menu icon function.
+ *
+ * Returns a base64 data URI for the SVG icon in the menu.
+ *
+ * @since 3.0
+ *
+ * @return string
+ */
+if ( ! function_exists( 'aioseop_get_menu_icon' ) ) {
+
+	function aioseop_get_menu_icon() {
+		$svg = '<svg version="1.1" xmlns="http://www.w3.org/2000/svg"
```

Comment:
Is it possible to make this an SVG file? If so, you might be able to eliminate this function by linking the SVG file directly.

Reply:
Has to be a base64-encoded SVG using a data URI which needs to begin with 'data:image/svg+xml;base64,' so I'm afraid that linking it directly is not possible (I tried).

Reply:
Ok, I was just curious if you had explored it.